### PR TITLE
3469 - 0

### PR DESCRIPTION
**Pull request by Morozov-Alexander**
## Номер
3469

## Номер задания
0

## Ссылка на видео с демо
https://youtu.be/-jjFY54evn0

### CODE REVIEW

**Comment by Vadser** [comment]
Please rewrite commits messages

**Comment by Vadser** [comment]
Please change the folder name, please read readme https://github.com/rubizza-camp/summer-2019

**Comment by Morozov-Alexander** [comment]
Commit history has been rewritten and the folder has been renamed.

File: 3469/0/triangle.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+def check_variable(tr_a, tr_b, tr_c)
+  return false if tr_a * tr_b * tr_c <= 0 || tr_a + tr_b <= tr_c || tr_a + tr_c <= tr_b || tr_c + tr_b <= tr_a
```
[the remark this answers is not in the document]
Comment (Morozov-Alexander) [reply 3469/0/triangle.rb]: Ok, i fix this.

**Pull request merged**